### DISCUSSION TITLE
installation problem

### DISCUSSION THREAD

**Discussion by ems-tuner** [collin80/SavvyCAN#480] [General]
can not install savvy can on 2022 kali linux , no such directory folder ,anyone can help?

**Comment by collin80** [comment]
It's probably an issue with some external dependency. The binary is packaged as an AppImage but still I believe it has external dependencies on things like glibc. So, my guess is that something just isn't where it thinks it should be or isn't the version it thinks it should be. You might try running it from the command line and seeing if there is anything specific being mentioned.
